tfhe/benches/integer/bench.rs

```diff
@@ -2146,6 +2158,91 @@ criterion_group!(
     unchecked_scalar_ge_parallelized,
 );
 
+//================================================================================
+//     Miscellaneous Benches
+//================================================================================
+
+fn bench_server_key_cast_function<F>(
+    c: &mut Criterion,
+    bench_name: &str,
+    display_name: &str,
+    cast_op: F,
+) where
+    F: Fn(&ServerKey, RadixCiphertext, usize),
+{
+    let mut bench_group = c.benchmark_group(bench_name);
+    bench_group
+        .sample_size(15)
+        .measurement_time(std::time::Duration::from_secs(30));
```

Comment:
aren't we at 10/60s on the other benches ?

Reply:
We do a lot of tests for this operations `bit_sizes_tested ** 2`, so I thought that reducing by two these tests would be beneficial in term of execution duration.

Reply:
yes indeed